src/exercises/FeedbackButtons.js

```diff
@@ -1,38 +1,136 @@
 import * as s from "./FeedbackButtons.sc.js";
-import { Link } from "react-router-dom";
+import { useState, useEffect } from "react";
 import strings from "../i18n/definitions";
 
-const buttons = [
-  { name: "Too Easy", value: "too_easy" },
-  { name: "Too Hard", value: "too_hard" },
-  { name: "Bad example", value: "not_a_good_example" },
-  { name: "Bad word", value: "bad_word" },
-  { name: "Other", value: "dont_show_it_to_me_again" },
-];
-export default function FeedbackButtons({ show, setShow, feedbackFunction }) {
-  function toggleShow() {
-    setShow(!show);
+export default function FeedbackButtons({
+  feedbackFunction,
+  currentExerciseType,
+  currentBookmarkToStudy,
+}) {
+  const buttons = [
+    { name: "Too Easy", value: "too_easy" },
+    { name: "Too Hard", value: "too_hard" },
+    { name: "Bad Translation", value: "bad_translation" },
+    { name: "Other", value: "other" },
+  ];
+
+  if (currentExerciseType !== "Match_three_L1W_to_three_L2W") {
+    buttons.splice(3, 0, { name: "Bad Context", value: "not_a_good_context" });
+  }
+  const [showInput, setShowInput] = useState(false);
+  const [className, setClassName] = useState("");
+  const [input, setInput] = useState("");
+  const [selectedId, setSelectedId] = useState();
+
+  useEffect(() => {
+    if (currentExerciseType !== "Match_three_L1W_to_three_L2W") {
+      setSelectedId(currentBookmarkToStudy.id);
+    }
+    console.log(selectedId);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  function isIterable(obj) {
```

Comment:
https://github.com/zeeguu-ecosystem/zeeguu-react/blob/development/README.md#6-general-functions-should-be-defined-in-general-modules-eg-utils

Reply:
Completely removed `isIterable`. See commit fca2e0c148b12bc6f78c8591952c1d99ac95ddfd